src/form/apo.jl

```diff
@@ -277,14 +288,18 @@ function constraint_storage_loss(pm::_PMs.AbstractActivePowerModel, n::Int, i, b
 end
 
 function constraint_mc_storage_on_off(pm::_PMs.AbstractActivePowerModel, n::Int, i, pmin, pmax, qmin, qmax, charge_ub, discharge_ub)
+    @assert all(pmin.<=pmax)
```

Comment:
Better to add this test at the template level and do it at once for all formulations?

Reply:
remove